src/legacy/core_plugins/console/public/src/directives/sense_settings.js

```diff
@@ -33,6 +35,7 @@ require('ui/modules')
         this.apply = () => {
           this.vals = settings.updateSettings(this.vals);
           $scope.kbnTopNav.close();
+          mappings.retrieveAutoCompleteInfo();
```

Comment:
Do you think it's worth leaving a comment here explaining what's going on?

```js
// When the user saves the settings, refetch them so they take effect immediately.
```

Reply:
good idea - updated w/ comment